opacus/data_loader.py

```diff
@@ -10,21 +10,18 @@
 def wrap_collate_with_empty(
     collate_fn: Optional[_collate_fn_t], sample_empty_shapes: Sequence
 ):
-    # TODO: does it work with packed sequences?
     def collate(batch):
         if len(batch) > 0:
             return collate_fn(batch)
         else:
-            return [torch.zeros(x) for x in sample_empty_shapes]
+            if type(sample_empty_shapes) is collections.abc.Sequence:
```

Comment:
This should be isinstance() instead. 
`type(list()) is collections.abc.Sequence` will return False
`isinstance(list(), collections.abc.Sequence)` will return True